routing/index_router.hpp

```diff
@@ -69,6 +69,19 @@ class IndexRouter : public IRouter
                             bool adjustToPrevRoute, RouterDelegate const & delegate,
                             Route & route) override;
 
+  /// \brief Finds the best segment (edge) which may be considered as the start of the finish of the route.
+  /// According to current implementation if a segment is near |point| and is almost codirectional
+  /// to |direction| vector, the segment will be better than others. If there's no an an almost codirectional
+  /// segment in neighbourhoods the closest segment to |point| will be chosen.
+  /// \param isOutgoing == true is |point| is considered as the start of the route.
+  /// isOutgoing == false is |point| is considered as the finish of the route.
+  /// \param bestSegmentIsAlmostCodirectional is filled with true if |bestSegment| is chosen
+  /// because vector |direction| and vector of |bestSegment| are almost equal and with false otherwise.
+  bool FindBestSegment(m2::PointD const & point, m2::PointD const & direction, bool isOutgoing,
```

Comment:
Missed comment about the returning value.

Reply:
I thought it obvious... 
Added.